streams/voxel_stream_file.cpp

```diff
@@ -79,6 +79,6 @@ void VoxelStreamFile::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_block_size"), &VoxelStreamFile::_get_block_size);
 
-	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "fallback_stream", PROPERTY_HINT_RESOURCE_TYPE, "VoxelStream"), "set_fallback_stream", "get_fallback_stream");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "fallback_stream", PROPERTY_HINT_RESOURCE_TYPE, "VoxelGeneratorImage,VoxelGeneratorNoise,VoxelGeneratorNoise2D,VoxelGeneratorTest,VoxelStreamBlockFiles,VoxelStreamRegionFiles"), "set_fallback_stream", "get_fallback_stream");
```

Comment:
Not having abstract classes such as `VoxelStream` or `VoxelGenerator` prevents the implementation of a custom one using a script. `VoxelStreamFile` is the abstract class for file-based streams.

Reply:
My gdscript demo extends VoxelStream with this configuration. This just removes it from the drop down list in the inspector. It doesn't affect the code, AFAIK and have seen.

Reply:
But then it means you need to set it via code? Or maybe you had it already in the property?

Reply:
Setting it in gdscript is what I mean. I don't see how there's any way to use the abstract classes in the inspector since they have no options or implementation. That's all that was removed: Useless options from the inspector.

In gdscript they can still be used, and it seems to be the only way to use them:

Extends VoxelStream:
https://github.com/tinmanjuggernaut/voxelgame/blob/master/project/fps_demo/scripts/MyStream.gd

Loads terrain based on custom VoxelStream:
https://github.com/tinmanjuggernaut/voxelgame/blob/master/project/fps_demo/scripts/CodeTerrain.gd

Reply:
Are you still concerned about this? With my current patches, the abstract classes are not visible in the inspector dropdown, but they are still accessible via gdscript and show up in the help. My demo extends VoxelGenerator just fine and it extended VoxelStream before that without issue.

Reply:
If you make a custom script (which will them inherit an abstract class) and want to assign it in the inspector, it is a valid option to create it on the property and assign the script, or even drag and drop an existing one.

Reply:
I made one and tried it out. Ok, I see. I have reverted these changes.